sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php

```diff
@@ -1181,6 +1251,37 @@ function loop_status($lcd) {
 					}
 					$updateSummary = false;
 					break;
+				case "scr_interfaces_link":
+					// We only want build_interface_link_list() to be
+					// called once per loop, and only if it's needed
+					if ($ifLinkList == null)
+						$ifLinkList = build_interface_link_list();
```

Comment:
The body of this `if` needs braces around it.

Reply:
all fixed